nrf_802154/serialization/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+# Platform needs paths to serialization headers
+target_include_directories(nrf-802154-platform
+  PRIVATE
+    spinel_serialization/zephyr/include
```

Comment:
What's the purpose of `spinel_serialization/zephyr` part in each path?
It suggests this module depends on Zephyr, but I hope it does not.

Reply:
I think it can be removed.